spec/features/brexit_checker/accounts_spec.rb

```diff
@@ -6,259 +6,237 @@
   include GdsApi::TestHelpers::AccountApi
   include GdsApi::TestHelpers::EmailAlertApi
 
-  before do
-    stub_request(:get, Plek.find("account-manager")).to_return(status: 200)
-  end
+  let(:mock_results) { %w[nationality-eu] }
 
-  context "with accounts enabled" do
-    before do
-      allow(Rails.configuration).to receive(:feature_flag_govuk_accounts).and_return(true)
+  context "the user is not logged in" do
+    context "/transition-check/results" do
+      it "shows the normal call-to-action" do
+        given_i_am_on_the_results_page
+        expect(page).to have_content(I18n.t("brexit_checker.results.email_sign_up_title"))
+      end
     end
 
-    let(:mock_results) { %w[nationality-eu] }
-
-    context "the user is not logged in" do
-      context "/transition-check/results" do
-        it "shows the normal call-to-action" do
-          given_i_am_on_the_results_page
-          expect(page).to have_content(I18n.t("brexit_checker.results.email_sign_up_title"))
-        end
+    context "/transition-check/saved-results" do
+      it "redirects to login page" do
+        given_i_am_on_the_saved_results_page
+        expect(page).to have_current_path(
+          "#{Plek.find('frontend')}/sign-in?#{CGI.escape('redirect_path=/transition-check/saved-results')}",
+        )
       end
+    end
 
-      context "/transition-check/saved-results" do
-        it "redirects to login page" do
-          stub_account_api_get_sign_in_url(redirect_path: "/transition-check/saved-results")
-          given_i_am_on_the_saved_results_page
-          expect(current_path).to eq(transition_checker_new_session_path)
-        end
+    context "/transition-check/edit-saved-results" do
+      it "redirects to login page" do
+        stub_account_api_get_sign_in_url(redirect_path: "/transition-check/saved-results")
```

Comment:
finder-frontend isn't calling account-api to sign in, so it doesn't need this stub.